bigquery/unit_tests/test_client.py

```diff
@@ -499,16 +499,7 @@ def test_run_sync_query(self):
 
 
 class _Credentials(object):
-
-    _scopes = None
-
-    @staticmethod
-    def create_scoped_required():
-        return True
-
-    def create_scoped(self, scope):
-        self._scopes = scope
-        return self
+    pass
```

Comment:
Can you just nuke this class and do a find-and-replace `_Credentials()` as `object()`?